sec_certs/certificate.py

```diff
@@ -130,7 +130,7 @@ class WebScan(ComplexSerializableType):
         algorithms: Optional[List[Dict[str, str]]]
         tested_conf: Optional[List[str]]
         description: Optional[str]
-        mentioned_certs: Optional[List[str]]
+        mentioned_certs: Optional[List[Dict]]
```

Comment:
would be nice to know the exact type of `Dict`, like `Dict[str, str]`

Reply:
oh, I realized this was completely unnecessary, thanks, will rewrite